astroquery/skyview/core.py

```diff
@@ -251,10 +251,10 @@ def get_image_list(self, position, survey, *, coordinates=None,
 
         if radius is not None:
             size_deg = str(radius.to(u.deg).value * 2)
-        elif width and height:
-            size_deg = "{0},{1}".format(width.to(u.deg).value,
-                                        height.to(u.deg).value)
-        elif width and height:
+        elif width is not None and height is not None:
```

Comment:
Shouldn't we instead check for Quantity inputs here? E.g. based on the docstring we are expecting Quantities or Nones, but not integers.

Reply:
the code allows float or integer arguments to be interpreted as degrees, which matches the skyview web api.  But I agree, it would be better to require a qty explicitly